docs/usage/help.rst.inc

```diff
@@ -23,7 +23,8 @@ store all files as `some/path/.../file.ext` and ``borg create
 File patterns support these styles: fnmatch, shell, regular expressions,
 path prefixes and path full-matches. By default, fnmatch is used for
 ``--exclude`` patterns and shell-style is used for the experimental
-``--pattern`` option.
+``--pattern`` option. For commands that support patterns in their ``PATH``
+argument like (``borg list``), the default pattern is path prefix.
```

Comment:
see 1st line of this file: this is autogenerated.

so you need to revert this change and do the respective change in `archiver.py`.

Reply:
Sorry, forgot.